docs/modules/jellyfish.md

```diff
@@ -0,0 +1,8 @@
+---
+Name: Jellyfish
+URL: http://www.cbcb.umd.edu/software/jellyfish/
+Description: >
+    JELLYFISH is a tool for fast, memory-efficient counting of k-mers in DNA.
+---
+
+JELLYFISH is a tool for fast, memory-efficient counting of k-mers in DNA. A k-mer is a substring of length k, and counting the occurrences of all such substrings is a central step in many analyses of DNA sequence. JELLYFISH can count k-mers using an order of magnitude less memory and an order of magnitude faster than other k-mer counting packages by using an efficient encoding of a hash table and by exploiting the "compare-and-swap" CPU instruction to increase parallelism.
```

Comment:
Need a description of how filenames need to be formatted here (and perhaps mention how to customise search patterns).